src/apps/concepts/api.ts

```diff
@@ -36,20 +36,48 @@ const api = {
       classFilters = [] as string[],
       sourceFilters = [] as string[],
       includeRetired = false
-    ) =>
-      authenticatedInstance.get(conceptsUrl, {
-        params: {
-          page,
-          limit,
-          q: buildPartialSearchQuery(q),
-          [sortDirection]: sortBy,
-          ...optionallyIncludeList("datatype", dataTypeFilters),
-          ...optionallyIncludeList("conceptClass", classFilters),
-          ...optionallyIncludeList("source", sourceFilters),
-          includeRetired,
-          timestamp: new Date().getTime()
+    ) => {
+        let params = {
+            page,
+            limit,
+            q: buildPartialSearchQuery(q),
+            [sortDirection]: sortBy,
+            ...optionallyIncludeList("datatype", dataTypeFilters),
+            ...optionallyIncludeList("conceptClass", classFilters),
+            ...optionallyIncludeList("source", sourceFilters),
+            timestamp: new Date().getTime()
+        };
+        if(includeRetired) {
+            // @ts-ignore
```

Comment:
Why are we ignoring types here? Any specific reason?

Reply:
Removed it